hardware-and-software-requirements.md

```diff
@@ -34,7 +34,7 @@ In v8.1 LTS, TiDB ensures multi-level quality standards for various combinations
         <td><ul><li>x86_64</li><li>ARM 64</li></ul></td>
       </tr>
       <tr>
-        <td><ul><li>Red Hat Enterprise Linux 7.3 or a later 7.x version</li><li>CentOS 7.3 or a later 7.x version</li></ul></td>
+        <td><ul><li>Red Hat Enterprise Linux 7.3 or a later 7.x version</li><li>CentOS 7.3 or a later 7.x version ⚠️</li></ul></td>
```

Comment:
Can we remove this ⚠️ icon as it is a universal symbol for a warning? Users might not understand what it means until reading the note after the table. Perhaps, we can update it to the following:

```suggestion
        <td><ul><li>Red Hat Enterprise Linux 7.3 or a later 7.x version</li><li>CentOS 7.3 or a later 7.x version (supported until TiDB v8.4.0) </li></ul></td>
```

Reply:
But this is only for LTS versions. 🤔